.golangci.yml

```diff
@@ -0,0 +1,34 @@
+issues:
+  exclude-rules:
+    - linters: gosimple
+      text: "S1002: should omit comparison to bool constant"
+  exclude-use-default: true
+  max-same-issues: 0
+  max-issues-per-linter: 0
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - goimports
+    - golint
+    - gosimple
```

Comment:
as mentioned in the other PR, if `staticcheck` is a superset of `gosimple`, let's remove the latter